Detectors/EMCAL/reconstruction/include/EMCALReconstruction/CTFCoder.h

```diff
@@ -156,8 +156,8 @@ void CTFCoder::decode(const CTF::base& ec, VTRG& trigVec, VCELL& cellVec)
       cellVec.emplace_back(cell);
       cellCount++;
     }
-    uint32_t trigBits = trigger[itrig];
-    trigVec.emplace_back(ir, trigBits, firstEntry, entries[itrig]);
+    auto currenttrigger = trigVec.emplace_back(ir, firstEntry, entries[itrig]);
+    currenttrigger.setTriggerBitsCompressed(trigger[itrig]);
```

Comment:
Reference for some reason seems to not work. The trigger bits stay unmodified. Will change it in the same way as done for the cells.

Reply:
you should use `auto& currenttrigger = trigVec.emplace_back(ir, firstEntry, entries[itrig])`